dist/index.html

```diff
@@ -1 +1 @@
-<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Document</title><script defer="defer" src="app.js"></script></head><body><h1>fasfdfsfsfd</h1></body></html>
\ No newline at end of file
+<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Webpack Todo List</title><link rel="stylesheet" href="style.css"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"><link href="assets/css/font-awesome.css" rel="stylesheet"/><script defer="defer" src="app.js"></script></head><body><section><div class="main-div"><p id="header">Today's To Do <i class="fas fa-sync"></i></p><form class="form"><input class="dataEntry" placeholder="Enter Task" required></form><p id="clear">Clear all completed</p></div></section></body></html>
```

Comment:
- [ ] Kindly make sure that the delete button is shown and apparent to the user even if the task is not marked as completed. This would enable the user to delete uncompleted tasks.

![image](https://user-images.githubusercontent.com/62290545/173352678-9b27f46c-0e8c-4b95-ae1c-386e9df99e53.png)